### PR TITLE
add link to docs contributor guide

### PR DESCRIPTION
**Pull request by mairaw**
Replaces PR #5426

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 Thank you for your interest in contributing to the .NET documentation!
 
+> We're in the process of moving our guidelines into a site-wide contribution guide.
```
Comment (jzeferino) [inline CONTRIBUTING.md]: should't be better to use `>>>` instead of `>` for each line?

**Review by jzeferino — APPROVED** [review]
LGTM

File: CONTRIBUTING.md (hunk 1 above)
Comment (svick) [reply CONTRIBUTING.md]: How would that help? I think that triple blockquote looks worse and doesn't make sense.

Compare:

>>> We're in the process of moving our guidelines into a site-wide contribution guide. 

With:

> We're in the process of moving our guidelines into a site-wide contribution guide.

Comment (jzeferino) [reply CONTRIBUTING.md]: This is my bad. I was confusing the markdown operator > here on GitHub with the >>> from Slack when we have multiples lines.

**Review by jzeferino — APPROVED** [review]
LGTM.

**Review by BillWagner — APPROVED** [review]
LGTM.  You can :shipit: when ready

**Pull request merged**